README.md

```diff
@@ -72,7 +74,9 @@ __OPTION 3:__ If you are going to deploy this in a **Kubernetes cluster**, check
 
 ### Setting up Prometheus Alert Manager
 
-By default, __prometheus-msteams__ creates a request uri handler __/alertmanager__.
+By default, __prometheus-msteams__ creates a request uri handler __/alertmanager__ and a dynamic handler __/webhook/*__.
+
+#### uri handler /alertmanager
```

Comment:
I'd suggest to phrase this `static uri handler` (as it could be anything else than `alertmanager`) and ...